src/OrchardCore.Modules/OrchardCore.Localization/ViewModels/LuceneSettingsViewModel.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Globalization;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace OrchardCore.Localization.ViewModels
+{
+    public class LocalizationSettingsViewModel
+    {
+        [BindNever]
+        public CultureEntry[] Cultures { get; set; } = Array.Empty<CultureEntry>();
+
+        public string SupportedCultures { get; set; }
```

Comment:
Is it not `string []`

Reply:
The name of the file should be LocalizationSettingsViewModel and I think it's a string because the Vue.js component returns a json string which contains multiple cultures. So yes normally it should be a string array but it's not required because of how we send the data in the form post.